Aptoide/aptoide-client-v8
# [ASV-2007] Implement Rakam

**What does this PR do?**

This PR aims at implementing Rakam and preparing it so it can log events in the future.
At the moment it is logging user properties (the same user properties that were being logged before and that are written in the ticket) and the events list is retuning a Collections.emptyList().
**This is only enabled on dev**, so please test that it is not being initialized on the other **build variants**. (ex prod, monetization, etc).

**PS:** We are using rakam specific tags for the user properties, hence why the var name has rakam on it. Because it is different from the other networks.
   
**Database changed?**

No

**Where should the reviewer start?**

- [ ] RakamEventLogger.java

**How should this be manually tested?**

  Run the app in dev flavour and check that you have logs regarding rakam.
Run the app in the other flavors and confirm that there is no logs regarding rakam and that everything works fine.

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-2007](https://aptoide.atlassian.net/browse/ASV-2007)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 

- RAKAM API KEY (already set in the repo we have for the keys).

**Code Review Checklist**
